Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -725,7 +727,10 @@ def _createOutputModuleInAddOutput(self, tier, streamType, eventContent, fileNam
         CppType='PoolOutputModule'
         if self._options.timeoutOutput:
             CppType='TimeoutPoolOutputModule'
-        if streamType=='DQM' and tier=='DQMIO': CppType='DQMRootOutputModule'
+        if streamType=='DQM' and tier=='DQMIO':
+            CppType='DQMRootOutputModule'
+            if len(fileName) > 6 and fileName[-6:] == '.rntpl':
+                fileName = fileName.replace('.rntpl', '.root')
```

Comment:
How about
```suggestion
            if fileName.endwith('.rntpl'):
                fileName = fileName.replace('.rntpl', '.root')
```
?

Although elsewhere the `replace()` is called unconditionally, so this could be
```suggestion
            fileName = fileName.replace('.rntpl', '.root')
```
as well.